suite-native/app/src/App.tsx

```diff
@@ -59,11 +62,12 @@ const AppComponent = () => {
 
     return (
         <FormatterProvider config={formattersConfig}>
-            <MessageSystemBannerRenderer />
+            {isOnboardingFinished && <OfflineHeader />}
+            {isOnboardingFinished && <MessageSystemBannerRenderer />}
             <RootStackNavigator />
             <ModalsRenderer />
             {/* NOTE: Rendered as last item so that it covers the whole app screen */}
-            <FeatureMessageScreen />
+            {isOnboardingFinished && <FeatureMessageScreen />}
```

Comment:
This should be visible even before onboarding is finished.

Reply:
22ed29dd69